src/clj/game/cards/resources.clj

```diff
@@ -2344,6 +2344,24 @@
                :effect (effect (update! (dissoc card :server-target)))}]
      :abilities [ability]}))
 
+(defcard "Smartware Distributor"
+  {:abilities [{:cost [:click 1]
+                :msg "place 3 [Credits]"
+                :effect (req (add-counter state side card :credit 3))}
+               (set-autoresolve :auto-distributor "take credit from Smartware Distributor")]
+   :events [{:event :runner-turn-begins
+             :optional
+             {:req (req (pos? (get-counters card :credit)))
+              :once :per-turn
+              :prompt "Use Smartware Distributor to gain 1 [Credits]?"
```

Comment:
I think this should be mandatory, given current text on card

Reply:
Excellent catch, thank you.